fact_extractor/plugins/unpacking/avm_kernel_image/install.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )" || exit
+
+echo "------------------------------------"
+echo "       install FREETZ tools         "
+echo "------------------------------------"
+
+sudo apt-get install -y bison flex gettext libtool-bin libtool libacl1-dev libcap-dev \
```

Comment:
We install freetz twice (see squashfs) So we should either join these plugins regarding setup or move the freetz installation to the common installation process

Reply:
Since squashfs is not related to AVM itself, we should move it to common.